### ISSUE TITLE
imageDragging: false option not working on FF

### ISSUE THREAD

**Issue by schnittchen** [yabwe/medium-editor#722]
I was able to drag&drop images into the editable even with imageDragging: false. I checkt both agains the minified and unminified version from the dist/ directory on master.

Here's my config:

```js
var initEditor = function(){
  var options = {
    toolbar: {
      buttons: ['bold', 'italic', 'underline', 'anchor', 'h2', 'h3', 'h4', 'removeFormat']
    },
    imageDragging: false,
    placeholder: { text: 'hier formatierten Text eingeben' }
  }
  new MediumEditor("#editable", options);
};
```

**Comment by j0k3r** [comment]
On which browser are you testing ?

I've setup a jsfiddle to reproduce your issue but it works for me: http://jsfiddle.net/j0k/jhvzzp1c/3/
 - Put your cursor into the first contenteditable, then drag an image, it works
 - Put your cursor into the second contenteditable, then drag an image, it doesn't works (actually it tells the browser to open the image)

Could you provide a jsfiddle to reproduce?

**Comment by schnittchen** [comment]
I can drag an image into both areas of your fiddle.

I'm using Iceweasle 38.0.1.

**Comment by j0k3r** [comment]
Could you try on a different browser ? I've no idea how _bad_ contenteditable is supported by Iceweasel (which I didn't know few seconds ago).

**Comment by schnittchen** [comment]
Same on FF 38.0.5 and 39.0 on windows (Iceweasle was on debian testing).
On IE 11.0.9.6, the tab is replaced with the image (file://...).

**Comment by j0k3r** [comment]
Ok, I've tried on Firefox 38.0.5 on my Mac (which seems to be almost the same as Iceweasel) and I can drag image on both areas too.
I previously tried on Chrome and it worked.

In fact, `imageDragging: false,` doesn't disable drag'n'drop feature, it does not enable our custom extension to handle dragging. Maybe, when imageDragging is disabled we should add some listeners on  `dragover` and `dragleave` event to cancel them..

**Comment by j0k3r** [comment]
This seems to work for me on Firefox:

```js
editor.subscribe('editableDrag', function(event){
    event.preventDefault();
    return false;
});
editor.subscribe('editableDrop', function(event){
    event.preventDefault();
    return false;
});
```

Could you try on Iceweasel too ?

**Comment by schnittchen** [comment]
With that snippet, dragging has no effect both with imageDragging == true and == false.

**Comment by j0k3r** [comment]
Yeh, sorry I didn't told you to test this behavior on `imageDragging: false`.
So this mean, this snippet should fix your problem. We just need to integrate it now.

**Comment by schnittchen** [comment]
Are you sure? I believe it breaks imageDragging: true.

**Comment by j0k3r** [comment]
Yeah of course, what I mean is if user defined `imageDragging: true` (or does nothing since imageDragging is enabled by default) we won't apply this snippet.
On the other hand, if `imageDragging: false` then we will apply this snippet.

**Comment by schnittchen** [comment]
Oh then that's fine :) It's too hot to think straight...

**Comment by nmielnik** [comment]
Interesting...so we think the image-dragging extension should go as far as to prevent default on all drag and drop events when `imageDragging: false`?

**Comment by j0k3r** [comment]
Well, there should be a case where if user disabled imageDragging, it might still want ability to drag text (could be a rare case but..).
Can we disable drag'n'drop only image but keep text ?

**Comment by nmielnik** [comment]
Fixed via #723
